ui/lib/core/addon/components/string-list.js

```diff
@@ -20,7 +20,6 @@ import { next } from '@ember/runloop';
  * @param {string} type=array - Optional type for inputValue.
  * @param {string} attrName - We use this to check the type so we can modify the tooltip content.
  * @param {string} subText - Text below the label.
- * @param {boolean} hideFormSection - If true do not add form-section class on surrounding div.
```

Comment:
Rather than using the `hideFormSection` arg is the intention to now apply the `is-box-shadowless` class directly to the component? I'm wondering if that will be unclear for anyone who hasn't worked with this component before.

Reply:
`hideFormSection` was added to remove the bottom shadow from string-list components within toggles
before: 

<img width="1378" alt="Screen Shot 2023-01-18 at 3 43 15 PM" src="https://user-images.githubusercontent.com/68122737/213319378-0eda6831-7548-441f-9ee7-defe53976d6c.png">

I think this goes back to needing to cleanup css 😂 It didn't seem like we didn't need to add a whole new argument here, and that instead we could add the existing css class to the stand-alone component by using splattributes. But this probably goes back to how we approach styling components.

Reply:
I prefer the class approach but was just looking for more context. The components that are used within `FormField` seem sensitive to change since they are so tightly coupled to the attr options but if we always want that shadow hidden within a form than this is a better approach.